tests/test_utils.py

```diff
@@ -9,7 +9,14 @@
                               iso_probability_contours_from_samples,
                               logsumexp, sample_compression_1d,
                               triangular_sample_compression_2d,
-                              insertion_p_value)
+                              insertion_p_value, compress_weights)
+
+
+def test_compress_weights():
+    w = compress_weights(w=np.ones(10), u=None)
+    assert_array_equal(w, np.ones(10))
+    w = compress_weights(w=None, u=np.random.rand(10))
+    assert_array_equal(w, np.ones(10))
```

Comment:
Always good to see new tests. Is there a specific reason we needed this for this PR?

Reply:
The usual issue with dropping a bunch of lines: the code coverage can drop causing the CI checks to fail. This test helps bring the coverage back up, such that CI checks pass.